extensions/rdw-kentekencheck/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "rdw-kentekencheck",
+  "title": "RDW Kentekencheck",
+  "description": "Shows details about Dutch cars by their license plate",
+  "icon": "list-icon.png",
+  "author": "jeffreyvanhees",
+  "categories": [
+    "Data"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "RDW Kentekencheck",
+      "subtitle": "Search details about Dutch cars by their license plate",
+      "description": "Shows details about Dutch cars by their license plate",
+      "mode": "view",
+      "icon": "command-icon.png"
```

Comment:
You can reuse list-icon.png here 🙂

Reply:
[Done!](https://github.com/raycast/extensions/pull/12689/commits/483d0c34bf615e82eb8249f4cd24dbbf612e7b1f)